src/Servers/HttpSys/src/RequestProcessing/RequestContext.cs

```diff
@@ -171,9 +171,11 @@ public void Abort()
             _disconnectToken = new CancellationToken(canceled: true);
         }
         ForceCancelRequest();
-        Request.Dispose();
+        // Request and/or Response can be null (even though the property doesn't say it can)
+        // if the constructor throws (can happen for invalid path format)
+        Request?.Dispose();
```

Comment:
Why not make the properties nullable?

Reply:
Because nullability is viral so every usage would need to be updated, and since these really shouldn't be null it'd be unnecessary to add null checks at every usage site. Ideally we would refactor the code more so these specific nullability checks aren't needed. But didn't want to do that as part of this fix.